src/VirtualClient/VirtualClient.Actions/Network/NetworkingWorkload/SockPerf/SockPerfMetricsParser.cs

```diff
@@ -121,7 +121,7 @@ private static void AddStatisticalMetrics(IList<Metric> metrics, List<double> pa
             metrics.Add(new Metric("Latency-P99.9", Statistics.QuantileCustom(packetsLatencyValues, 1d - 0.001d, QuantileDefinition.R3), MetricUnit.Microseconds, MetricRelativity.LowerIsBetter));
             metrics.Add(new Metric("Latency-P99.99", Statistics.QuantileCustom(packetsLatencyValues, 1d - 0.0001d, QuantileDefinition.R3), MetricUnit.Microseconds, MetricRelativity.LowerIsBetter));
             metrics.Add(new Metric("Latency-P99.999", Statistics.QuantileCustom(packetsLatencyValues, 1d - 0.00001d, QuantileDefinition.R3), MetricUnit.Microseconds, MetricRelativity.LowerIsBetter));
-            metrics.Add(new Metric("Latency-Mad", Stats.MedianAbsoluteDeviation(packetsLatencyValues.ToArray()), MetricUnit.Microseconds, MetricRelativity.LowerIsBetter));
+            metrics.Add(new Metric("Latency-Mad", Extreme.Statistics.Stats.MedianAbsoluteDeviation(packetsLatencyValues.ToArray()), MetricUnit.Microseconds, MetricRelativity.LowerIsBetter));
```

Comment:
Added specific Namespace to avoid conflicts with the Stats Class in [BlenderBenchmarkResult](https://github.com/microsoft/VirtualClient/blob/c05222808d4af9fc4b3f1a3579a844b5ccce5830/src/VirtualClient/VirtualClient.Actions/BlenderBenchmark/BlenderBenchmarkResult.cs#L280)